model/src/main/kotlin/ScannerDetails.kt

```diff
@@ -37,4 +39,19 @@ data class ScannerDetails(
          * The configuration of the scanner, could be command line arguments for example.
          */
         val configuration: String
-)
+) {
+    private val incompatibleVersionDiffs = listOf(
```

Comment:
Is it really worth to have this property? Personally, I find it more readable if it was inlined. Esp. as the name is also a bit awkward. Anyway, approved, I'll leave this to you.

Reply:
Noticed your comment only after the merge, yes, unnecessary object creation should be avoided, do you have an idea for a better name?

Reply:
Not really... maybe `unacceptable` instead of `incompatible`? Or just keep it as it is.